src/components/forms/BasicInfoForm.js

```diff
@@ -206,14 +206,7 @@ export default compose(
     setLocation,
   }) => (
     <Fragment>
-      <div
-        css={`
-          ${theme.row};
-          margin-bottom: 20px;
-          z-index: 1;
-          position: relative;
-        `}
-      >
+      <Row mb="20px" zIndex="1" position="relative" minHeight="0">
```

Comment:
I suggest we should consider enforcing styled components on anything that has more than 3 inline styles :P